<launch>

	<!-- usb_cam: RUNS usb CAM -->
	<node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
		<param name="video_device" value="/dev/video0" />
		<param name="image_width" value="640" /> <!-- 640 -->
		<param name="image_height" value="480" /> <!-- 480 -->
		<param name="pixel_format" value="yuyv" />
		<param name="camera_frame_id" value="usb_cam" />
		<param name="io_method" value="mmap"/>
	</node>

	

	<!-- Makes yolo detection node -->
	<node type="yolo_v2_ros_node" name="yolo" pkg="yolo_v2_ros" respawn="true" output="screen">

		<param name="basedir" value="$(find yolo_v2_ros)" />
		<param name="model_cfg" value="/cfg/yolo-voc.cfg" />
		<param name="weightfile" value="/weight/yolo-voc.weights" />
		<param name="datafile" value="/cfg/voc.data" />

		<param name="visualize_detections" value="true" />

		<param name="threshold" value="0.40" />
		<param name="publish_image" value="True" />
		<param name="topic_name" value="/usb_cam/image_raw" />
		<!-- "/Ptgrey_cameras/right/image_raw", "/camera/image_color", "/multisense/left/image_color", "/usb_cam/image_raw"    -->
	</node>
	
 

</launch>

